src/main/java/org/openstreetmap/atlas/geography/atlas/multi/MultiAtlas.java

```diff
@@ -516,7 +516,8 @@ public Area area(final long identifier)
     @Override
     public Iterable<Area> areas()
     {
-        return Iterables.translate(this.areaIdentifierToAtlasIndex, identifier -> area(identifier));
+        return Iterables.translate(this.areaIdentifierToAtlasIndices,
+                identifier -> area(identifier));
```

Comment:
Nit: You can shorten this slightly using Java 8's: `this::area`. Up to you!

Reply:
I'll do that. I actually didn't write that code, git just picked it up because I edited the name of the MultiMap. Looks like a good improvement though, so changed.